examples/C/H5D/CMakeLists.txt

```diff
@@ -9,12 +9,7 @@ include (C_sourcefiles.cmake)
 foreach (example ${examples})
   get_filename_component (example_name ${example} NAME_WE)
   add_executable (${EXAMPLE_VARNAME}_${example_name} ${PROJECT_SOURCE_DIR}/${example})
-  target_compile_options(${EXAMPLE_VARNAME}_${example_name}
```

Comment:
Are theses options supported in a different way?

Reply:
if I add it I get the error:

CMake Error at examples/C/H5D/CMakeLists.txt:12 (target_compile_options):
  target_compile_options called with incorrect number of arguments

Reply:
That means a variable is undefined